packages/onft-evm/package.json

```diff
@@ -0,0 +1,83 @@
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.cjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "artifacts/contracts/**/!(*.dbg).json",
+    "contracts/**/*"
+  ],
+  "scripts": {
+    "build": "forge build",
+    "build-contracts": "forge build && $npm_execpath hardhat compile --no-typechain --no-copy-artifacts && $npm_execpath hardhat size-contracts --no-compile",
+    "build-hardhat": "$npm_execpath hardhat compile",
+    "coverage:foundry": "forge coverage",
+    "debugpublish": "$npm_execpath publish --registry http://localhost:4873/",
+    "test": "forge test",
+    "test-contracts": "forge test && $npm_execpath hardhat test --no-compile",
+    "test-static-analyze": "$npm_execpath hardhat clean && $npm_execpath build-hardhat && python3 -m slither evm --ignore-compile --compile-force-framework hardhat --npx-disable",
+    "test_onft721_batch_0": "forge test --match-test test_onft721_batch_zero --gas-report",
+    "test_onft721_batch_1": "forge test --match-test test_onft721_batch_one --gas-report",
+    "test_onft721_batch_10": "forge test --match-test test_onft721_batch_ten --gas-report",
+    "test_onft721_batch_100": "forge test --match-test test_onft721_batch_hundred --gas-report",
+    "test_onft721_batch_250": "forge test --match-test test_onft721_batch_two_hundred_fifty --gas-report",
+    "test_onft721_batch_3": "forge test --match-test test_onft721_batch_three --gas-report",
+    "test_onft721_batch_only_2": "forge test --match-test test_onft721_batch_only_two --gas-report",
+    "test_onft721_double": "forge test --match-test test_onft721_double --gas-report",
+    "test_onft721_single": "forge test --match-test test_onft721_single --gas-report",
+    "test_onft721_triple": "forge test --match-test test_onft721_triple --gas-report"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.9",
+    "@layerzerolabs/eslint-config-next": "~2.3.3",
+    "@layerzerolabs/lz-definitions": "^2.3.3",
+    "@layerzerolabs/lz-evm-messagelib-v2": "^2.3.3",
+    "@layerzerolabs/lz-evm-oapp-v2": "^2.3.3",
+    "@layerzerolabs/lz-evm-protocol-v2": "^2.3.3",
+    "@layerzerolabs/lz-evm-v1-0.7": "^2.3.3",
+    "@layerzerolabs/lz-v2-utilities": "^2.3.3",
+    "@layerzerolabs/prettier-config-next": "^2.3.3",
+    "@layerzerolabs/solhint-config": "^2.3.3",
+    "@layerzerolabs/test-devtools-evm-foundry": "~0.2.3",
+    "@layerzerolabs/toolbox-foundry": "~0.1.5",
+    "@layerzerolabs/toolbox-hardhat": "~0.2.21",
+    "@nomicfoundation/hardhat-ethers": "^3.0.5",
+    "@nomiclabs/hardhat-ethers": "^2.2.3",
+    "@openzeppelin/contracts": "^5.0.2",
+    "@openzeppelin/contracts-upgradeable": "^5.0.2",
+    "@rushstack/eslint-patch": "^1.7.0",
+    "@types/chai": "^4.3.11",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "~18.18.14",
+    "chai": "^4.4.1",
+    "dotenv": "^16.4.1",
+    "erc721a": "https://github.com/chiru-labs/ERC721A",
```

Comment:
This looks sketchy, i'd imagine some git commit ref here since when somebody installs this, they will just get whatever is on main now